dashboard/app/models/sections/section.rb

```diff
@@ -193,7 +193,16 @@ def summarize
       linkToAssigned: link_to_assigned,
       numberOfStudents: students.length,
       linkToStudents: "#{base_url}#{id}/manage",
-      sectionCode: code
+      code: code,
+      stageExtras: stage_extras,
+      pairingAllowed: pairing_allowed,
+      loginType: login_type,
+      courseId: course_id,
+      script: {
+        id: script_id,
+        name: script.try(:name),
```

Comment:
Do we use script name anywhere? (i.e. could we get away with this just being scriptId)?